conf/app.routes

```diff
@@ -50,8 +50,8 @@ GET         /ead-print-view/:mrn                           controllers.EADContro
 
 # Cancel declaration
 
-GET         /cancel-declaration                            controllers.CancelDeclarationController.displayPage
-POST        /cancel-declaration                            controllers.CancelDeclarationController.onSubmit
+GET         /cancel-declaration                            controllers.CancelDeclarationController.displayPage(additionalDecType: String)
```

Comment:
I'd rather not pass this `additionalDecType` parameter on the query string unchecked. Can you instead add to the CancelDeclarationController the `JourneyAction` and grab the `additionalDecType` from the cache. This will prevent the (granted unlikely) possibility of user's re-writing the url and sticking in some rubbish query string value.